health-monitors/gpu-health-monitor/gpu_health_monitor/tests/test_healthz.py

```
@@ -0,0 +1,77 @@
+        self.socket = _FakeSocket()
+        self.bound = False
+        self.server_bind()
+
+    def server_bind(self):
+        self.bound = True
+
+    def serve_forever(self):
+        pass
+
+
+@pytest.mark.parametrize("addr", ["", "0.0.0.0"])
```

Ruff (0.16.0)

[error] Possible binding to all interfaces

(S104)